client/app/services/intercom.js

```diff
@@ -1,13 +1,18 @@
 import Ember from 'ember';
 import config from 'client/config/environment';
 
+/* eslint-disable new-cap*/
 export default Ember.Service.extend({
-  intercom: window.Intercom || Ember.K,
+  init() {
+    this._super(...arguments);
+    if (config.intercom.disabled) {
+      window.Intercom = Ember.K;
```

Comment:
It wasn't consistently updating the `intercom` attribute when the keys for intercom were set in .env. I first tried to reference the attribute with  `this.get('intercom')` but that did not work so used this workaround.